content/doc/book/glossary/index.adoc

```diff
@@ -129,15 +133,16 @@ Workspace:: [[workspace]]
     <<pipeline,Pipeline>> run completes unless specific Workspace cleanup policies
     have been put in place on the Jenkins <<master,Master>>.
 
-
-////
-XXX: It's currently unclear to me (rtyler) whether these merit definition
-
-== Project/Pipeline Status
+[build-status]
+== Build Status
 
 Aborted:: [[aborted]]
+    The <<build,Build>> was manually aborted.
```

Comment:
It can also be aborted automatically such as in the case of timeouts

Reply:
Definition changed in latest commit.